lua/kubectl/utils/viewsTable.lua

```diff
@@ -16,6 +16,8 @@ return {
   services = { "services", "service", "svc" },
   configmaps = { "configmaps", "configmap", "cm" },
   crds = { "customresourcedefinitions", "crds", "crd", "customresourcedefinitions.apiextensions.k8s.io" },
+  namespace = { "namespace", "namespaces", "ns" },
```

Comment:
While I like the idea of adding namespaces here, that would remove it from the fallback views and we wouldn't be able to edit/delete them

Reply:
You are right! i'll remove it